homeassistant/components/pjlink/media_player.py

```diff
@@ -104,6 +162,10 @@ def _setup_projector(self):
             with self.projector() as projector:
                 if not self._attr_name:
                     self._attr_name = projector.get_name()
+                if not self._manufacturer:
+                    self._manufacturer = projector.get_manufacturer()
+                if not self._model:
+                    self._model = projector.get_product_name()
```

Comment:
You should instead set `self._attr_device_info`:
```suggestion
                manufacturer = projector.get_manufacturer()
                model = projector.get_product_name()`
                self._attr_device_info = DeviceInfo(...)
```